windows.js

```diff
@@ -6,9 +6,14 @@ const TEN_MEBIBYTE = 1024 * 1024 * 10;
 
 export default async function killTabs() {
 	const command = 'wmic process where Caption=\'chrome.exe\' get CommandLine,ProcessId /format:list';
-	const stdout = await promisify(childProcess.exec)(command, {maxBuffer: TEN_MEBIBYTE});
+	const response = await promisify(childProcess.exec)(command, {maxBuffer: TEN_MEBIBYTE});
 
-	return execall(/CommandLine=(.+)\s+ProcessId=(\d+)/g, stdout).map(element => ({
+	if (response.stderr) {
+		console.error(response.stderr);
+		throw new Error('Failed to kill tabs.');
```

Comment:
Can you pass `stderr` (wrapped in an error) as the `Error#cause`?

https://2ality.com/2021/06/error-cause.html

Reply:
you mean like

```js
if (response.stderr) {
    throw new Error(response.stderr);
}
```

Reply:
No, please re-read my comment and check out the link.

Reply:
From my understanding we wrap the existing error message by `stderr` into another error. The new error define as  `#cause`  the message from `stderr`. So something like the following if I got it correctly??

```js
throw new Error(`Failed to kill tabs.`, {cause: new Error(stderr)});
```

Reply:
Correct